src/main/java/com/sericulture/authentication/model/entity/RoleMaster.java

```diff
@@ -0,0 +1,40 @@
+package com.sericulture.authentication.model.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.util.Date;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.util.Date' import. Should be before 'org.hibernate.annotations.UpdateTimestamp'.